packages/hyperion-autologging/src/ALInteractableDOMElement.ts

```diff
@@ -76,15 +77,34 @@ function elementHasEventHandler(node: HTMLElementWithHandlers, eventName: HTMLEl
 }
 
 function ignoreInteractiveElement(node: HTMLElement) {
-  const innerHeight: number = window.innerHeight;
-  const innerWidth: number = window.innerWidth;
   return (
     node.tagName === 'BODY' ||
-    node.tagName === 'HTML' ||
-    (node.clientHeight === innerHeight && node.clientWidth === innerWidth)
+    node.tagName === 'HTML'
   );
 }
 
+const isFunctionBodyNoop = (body: string): boolean => {
+  // Extract body between { } via regex and trim to determine noop
+  const match = body.match(/{(?<body>(.*))}/);
+  return match?.groups?.body?.trim() === '';
```

Comment:
The following should be faster and safer:
`return /^[^(]*\([^)]*\)\s*\{\s*\}/.test(body);`

Reply:
Will update,  but that doesn't work for `() => {}`.
```
> const x = () => {}
undefined
> /^[^(]*\([^)]*\)\s*\{(\s*| return; | void 0; )\}/.test((() => {}).toString())
false
> x.toString()
'() => {}'
> x()
undefined
```

Reply:
Yes, I missed that. Your updated regex seems will do it.